test.sh

```diff
@@ -5,13 +5,7 @@ set -e
 # run only on pull requests and when NODE_ENV=test
 if [[ ${CI} != "true" || (${NODE_ENV} = "test" && ${TRAVIS_PULL_REQUEST} != "false") ]]; then
   # set path for test directory
-
-  # setup to run browser GUI
-  if [[ ${CI} = "true" ]]; then
-    export DISPLAY=:99.0
-    sh -e /etc/init.d/xvfb start &
-    sleep 3 # give xvfb some time to start
-  fi
+  bundler --version
```

Comment:
Is this doing anything? Is the comment above still valid?

Reply:
the comment above is leftover from before, not from now. but I can remove it.
the bundler --version is a quick log to let us know which version travis is running. I think is a good idea to leave it